src/extensions/default/AutoUpdate/main.js

```diff
@@ -337,6 +337,19 @@ define(function (require, exports, module) {
                     installerName = "Brackets." + buildName.split(" ").join(".") + ext,
                     downloadURL;
 
+                //AUTOUPDATE_PRERELEASE BEGIN
+                // This code change is needed for Auto-update support in prereleases
+                // And will be removed eventually for stable releases
+                // Overwriting the tag and downloadURL for prereleases
+                {
+                    if(buildName && buildName.indexOf("Pre Release") !== -1) {
```

Comment:
Ideally we shouldn't do this kind of index of check. This are prone to failures because of mixed cases. We can introduce a prerelease flag in the release entry and just put a check for that. `buildName` is being constructed with `buildNumber` and `versionString` I guess. Don't put spaces in the value.